### PR TITLE
feat(app): exclude windows terminal attach

### PR DESCRIPTION
**Pull request by Frigyes06**

### CODE REVIEW

**Comment by Frigyes06** [comment]
I will eat my hat

File: src/main.rs [hunk 1]
```diff
@@ -26,8 +26,17 @@ fn main() -> iced::Result {
     gui::UadGui::start()
 }
 
+/// Sets up logging to a new file in CACHE_DIR/UAD_{time}.log
+/// Also attaches the terminalon windows machines
```
Comment (edgy-b) [inline src/main.rs:30]: Small typo mistake `terminal on`

**Review by edgy-b — APPROVED** [review]

File: src/main.rs [hunk 2]
```diff
@@ -26,8 +26,17 @@ fn main() -> iced::Result {
...
 }
 
+/// Sets up logging to a new file in CACHE_DIR/UAD_{time}.log
+/// Also attaches the terminal on Windows machines
+/// '''
+/// match setup_logger().expect("Error setting up logger")
+/// '''
 pub fn setup_logger() -> Result<(), fern::InitError> {
-    attach_windows_console();
+    /// Attach Windows terminal, only on windows
```
Comment (AnonymousWP) [inline src/main.rs:35]:
```suggestion
    /// Attach Windows terminal, only on Windows
```

File: src/main.rs [hunk 3]
```diff
@@ -80,13 +89,23 @@ pub fn setup_logger() -> Result<(), fern::InitError> {
 /// (Windows) Allow the application to display logs to the terminal
 /// regardless if it was compiled with `windows_subsystem = "windows"`.
 ///
-/// This is a no-op when compiled to non-windows targets.
+/// This is excluded on non-windows targets.
```
Comment (AnonymousWP) [inline src/main.rs:92]:
```suggestion
/// This is excluded on non-Windows targets.
```

**Review by AnonymousWP — CHANGES_REQUESTED** [review]

**Review by AnonymousWP — APPROVED** [review]

**Comment by AnonymousWP** [comment]
Approved & merged, but if you could add some small explanation in the description, then that'd be great.

**Pull request merged**